dev-go/delve/delve-1.6.0.ebuild

```diff
@@ -1,25 +1,32 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 inherit go-module
+
 EGO_PN="github.com/go-delve/delve"
 SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
 KEYWORDS="~amd64"
 
 DESCRIPTION="A source-level debugger for the Go programming language"
 HOMEPAGE="https://github.com/go-delve/delve"
 
 LICENSE="MIT BSD BSD-2 Apache-2.0"
 SLOT="0"
+IUSE="test"
+RESTRICT="!test? ( test )"
 
-RESTRICT="test"
+DEPEND="dev-lang/go:="
```

Comment:
Actually delve is a go debugger so I thought is quite obvious have it as depend for both build an and functioning

Reply:
This is handled by *go-module.eclass* anyway.